src/modules/scores.js

```diff
@@ -0,0 +1,32 @@
+const userScores = [
```

Comment:
- [ ] [optiona] It would be nice if all your scores were in the API, then you wouldn't require this array to store your scores.